extensions/mssql/src/objectManagement/commands.ts

```diff
@@ -22,10 +22,11 @@ import { ServerRoleDialog } from './ui/serverRoleDialog';
 import { DatabaseRoleDialog } from './ui/databaseRoleDialog';
 import { ApplicationRoleDialog } from './ui/applicationRoleDialog';
 import { DatabaseDialog } from './ui/databaseDialog';
+import { ServerPropertiesDialog } from './ui/serverPropertiesDialog';
 
 export function registerObjectManagementCommands(appContext: AppContext) {
 	// Notes: Change the second parameter to false to use the actual object management service.
-	const service = getObjectManagementService(appContext, false);
+	const service = getObjectManagementService(appContext, true);
```

Comment:
@alanrenmsft You said this should be reverted, right? @barbaravaldez please confirm with him before merging in

Reply:
oh, no, please revert this, to false, also, @barbaravaldez this made me wonder whether you have actually tested the code with the actual STS backend, please revert and redo the testing. :)

Reply:
yup, let me revert. Thank you!